src/firebase/firebase.js

```diff
@@ -34,10 +34,7 @@ const db = firebase.database(app);
  */
 export function addSong(songInfo, playlistName) {
   const dbRef = db.ref(
-    "playlists/" +
-      playlistName +
-      "/playlist/song-" +
-      Math.round(Math.random() * 1000)
```

Comment:
Remove this random int assignment since we're already doing it elsewhere in the codebase

Reply:
line 41 in `AddSong.js`